src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,85 @@
+import classNames from "classnames";
+import React from "react";
+import type { ClassName, ValueOf } from "types";
+
+export const BadgeType = {
+  ROUNDED_LARGE_NUMBER: "ROUNDED_LARGE_NUMBER",
+  UNDEFINED_LARGE_NUMBER: "UNDEFINED_LARGE_NUMBER",
+} as const;
+
+/**
+ * The props for the Badge component.
+ */
+export type Props = {
+  /**
+   * Numeric value to be displayed.
+   */
+  value: number;
+  /**
+   * The type of the badge component.
+   */
+  badgeType?: ValueOf<typeof BadgeType> | string;
```

Comment:
Do we need to allow any string here? I guess it should be limited to the allowed values only?

Reply:
Right, we dont need it.